cmake/core-files.cmake

```diff
@@ -531,6 +531,7 @@ set(MBGL_CORE_FILES
     include/mbgl/util/immutable.hpp
     include/mbgl/util/indexed_tuple.hpp
     include/mbgl/util/interpolate.hpp
+    include/mbgl/util/io.hpp
```

Comment:
Could we please retain that as private? We could use some other way of reading the file in GLFW.